src/Components/UI/NavBar/NavWrap.styles.js

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components';
+
+const NavWrap = styled.section`
```

Comment:
I think it would be good to use a `styled.nav` instead of the section.

Reply:
LGTM 🚀